### ISSUE TITLE
Neon JS cannot deserialize empty keys of NEO dictionaries

### ISSUE THREAD

**Issue by saltyskip** [CityOfZion/neon-js#397]
Hex String
```
82020007746f6b656e2f028204000269640201020003757269003a68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f7854694e30434e48676f526631486137434d2f67697068792e676966000a70726f7065727469657300074f472050455045000c727770726f7065727469657300000007746f6b656e2f038204000269640201030003757269003a68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f7854694e30434e48676f526631486137434d2f67697068792e676966000a70726f7065727469657300084f3320526f636b73000c727770726f706572746965730000
```

In plain text

```
token/iduri:https://media.giphy.com/media/xTiN0CNHgoRf1Ha7CM/giphy.gif
propertiesOG PEPErwpropertiestoken/iduri:https://media.giphy.com/media/xTiN0CNHgoRf1Ha7CM/giphy.gif
propertiesO3 Rocksrwproperties
```

This string contains a list of Non Fungible Tokens

The structure of the token object is 

`
{
 id: Int,
 uri: ByteArray,
 properties: ByteArray,
rwProperties: ByteArray
}
`

The rwProperties key should be "" or an empty string. NeonJS fails to deserialize this object because of the value of rwProperties is an empty string. It should be able to properly parse this object

using ```Neon.sc.StackItem.deserialize```

**Comment by hal0x2328** [comment]
@saltyskip It is possible the recently merged PR #394 fixes this issue?

**Comment by snowypowers** [comment]
I just tried it with v4.4.0 which has the fix. It does seem to deserialize it but your plain text string isnt human readable so im unable to visually compare it. Here is a screenshot instead.
![image](https://user-images.githubusercontent.com/9101245/55371363-d4a4e900-54b2-11e9-8009-1189aef5fa9c.png)
